wiki/osu!api/de.md

```diff
@@ -1,16 +1,15 @@
----
-outdated: true
-outdated_since: f90e775380f1c7041a62f3d7f5e3e29ba4326dee
----
-
 # osu!api
 
 osu! hat eine öffentliche API, die Entwickler nutzen können, um die osu! Plattform zu unterstützen und zu erweitern. Früher war sie eher privat und konnte nur auf Anfrage genutzt werden, mittlerweile kann sie allerdings jeder nutzen der über einen osu! Account verfügt.
 
-Du kannst eine Dokumentation von der API auf der [Wikiseite der osu!api](https://github.com/ppy/osu-api/wiki) finden. Die aktuelle Version der API nutzt private Keys für die Authentifizierung der Anfragen. Du kannst dir einen API Key auf der [Beantragungsseite](https://osu.ppy.sh/p/api) besorgen.
+Du kannst eine Dokumentation von der API auf der [Wikiseite der osu!api](https://github.com/ppy/osu-api/wiki) finden. Die aktuelle Version der API nutzt private Keys für die Authentifizierung der Anfragen. Du kannst dir einen API Key auf der [Beantragungsseite](https://osu.ppy.sh/p/api) besorgen. Denke aber daran, dass die Dokumentation der osu!api eventuell nicht vollständig oder umfangreich ist, und die API nicht aktiv entwickelt wird.
+
+---
+
+Version 2 der osu!api ist gerade in Arbeit und zu finden im [osu-web](https://github.com/ppy/osu-web). Die Dokumentation (keine Garantie, dass diese vollständig ist) findet sich auf ihrer [Referenzseite](https://docs.ppy.sh).
 
-Denke aber daran, dass die Dokumentation der osu!api eventuell nicht vollständig oder umfangreich ist, und die API nicht aktiv entwickelt wird. Eine neue und bessere API ist gerade in Arbeit und zu finden im [osu-web](https://github.com/ppy/osu-web).
+Die Authentifizierung für die neue API folgt dem [OAuth-2-Protokoll](https://oauth.net/2/). Clients können auf der [Einstellungsseite](https://osu.ppy.sh/home/account/edit) verwaltet werden.
```

Comment:
```suggestion
Die Authentifizierung für die neue API folgt dem [OAuth-2-Protokoll](https://oauth.net/2/). Clients können auf der [Einstellungen Seite](https://osu.ppy.sh/home/account/edit) verwaltet werden.
```
page itself is called `Einstellungen` in german so might consider using this instead but either is fine imo

Reply:
How about `auf der Seite "Einstellungen"`?

Reply:
that sounds even better